posthog/api/feature_flag.py

```diff
@@ -759,6 +759,23 @@ def my_flags(self, request: request.Request, **kwargs):
         methods=["GET"], detail=False, throttle_classes=[FeatureFlagThrottle], required_scopes=["feature_flag:read"]
     )
     def local_evaluation(self, request: request.Request, **kwargs):
+        # Check if team is quota limited for feature flags
+        if settings.DECIDE_FEATURE_FLAG_QUOTA_CHECK:
+            from ee.billing.quota_limiting import QuotaLimitingCaches, QuotaResource, list_limited_team_attributes
+
+            limited_tokens_flags = list_limited_team_attributes(
+                QuotaResource.FEATURE_FLAGS, QuotaLimitingCaches.QUOTA_LIMITER_CACHE_KEY
+            )
+            if self.team.api_token in limited_tokens_flags:
+                return Response(
```

Comment:
Should we be responding with an error here so they know they are being limited?

Reply:
yeah I was on the fence about this too – I know we can't fail decide because of quota limiting and instead have to just return a different permutation of the decide response, maybe with local eval I can be more fail-y – I haven't really looked at the SDKs to see how they handle it but we already are pretty happy to fail local eval with a 429 if we hit the customer rate limits, perhaps it's worth doing failing for a quota limit, too.  Good take.  Let me try this and see how it feels.